README.md

```diff
@@ -67,3 +67,34 @@ to handle output from:
 * Quantum Mechanics applications, such as Jaguar
 * Protein-Protein Docking applications
 * Many other backends used in both Life and Material Sciences
+
+Installation
+============
+
+Command line installation on a
+[Unix-like](https://en.wikipedia.org/wiki/Unix-like)
+operating system follows a typical configure, build, and test procedure.
+Configuration is via [CMake](https://en.wikipedia.org/wiki/CMake);
+building is via [Make](https://en.wikipedia.org/wiki/Make_(software));
+and testing can be performed by running a custom built program.  Here is an
+example command sequence:
```

Comment:
CMake can be instructed to  generate the build instructions for different build generators (see the `-G` option in the CMake documentation: https://cmake.org/cmake/help/latest/manual/cmake.1.html). `make` is just the default option (e.g., `make` won't be available on windows systems).


```suggestion
and testing can be performed by running a custom built program.  Here is an
example command sequence building via [Make](https://en.wikipedia.org/wiki/Make_(software)) and the `gcc` compiler:
```